Source/Scene/Scene.js

```diff
@@ -1444,6 +1450,55 @@ define([
             }
         },
 
+        /**
+         * The value used for gamma correction.
+         * @memberof Scene.prototype
+         * @type {Number}
+         * @default 2.2
+         */
+        gamma : {
+            get : function() {
+                return this._context.uniformState.gamma;
+            },
+            set : function(value) {
+                this._context.uniformState.gamma = value;
+            }
+        },
+
+        /**
+         * Whether or not to use high dynamic range rendering.
+         * @memberof Scene.prototype
+         * @type {Boolean}
+         * @default true
+         */
+        highDynamicRange : {
+            get : function() {
+                return this._hdr;
+            },
+            set : function(value) {
+                var context = this._context;
+                var hdr = value && context.depthTexture && (context.colorBufferFloat || context.colorBufferHalfFloat);
+                this._hdrDirty = hdr !== this._hdr;
+                this._hdr = hdr;
+            }
+        },
+
+        /**
+         * Gets or sets the color of the light emitted by the sun.
+         *
+         * @memberof Scene.prototype
+         * @type {Color}
+         * @default Color(0.8, 0.8, 0.8)
```

Comment:
Update the default to `(0.8, 0.85, 1.0)`

Reply:
Updated. @ggetz are we sure we don't want to use this value? https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/processPbrMaterials.js#L620